### PR TITLE
Change la pixel on recipe page to use monitor recipe

### PR DESCRIPTION
**Pull request by pablothedude**

### CODE REVIEW

File: src/AppBundle/Repository/RecipeRepository.php [hunk 1]
```diff
@@ -30,4 +31,23 @@ public function findForUserByCourse(User $user, Course $course)
 
         return array_column($results, 'id', 'id');
     }
+
+    /**
+     * @param string $slug
+     * @return Recipe
```
Comment (MKodde) [inline src/AppBundle/Repository/RecipeRepository.php]: Perhaps indicate null is an optional return type. Even tho the find method name already implies this.

File: app/DoctrineMigrations/Version20180117123823.php [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    public function down(Schema $schema)
+    {
+        // this down() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->addSql('ALTER TABLE recipe DROP slug');
+    }
+
+
+    public function postUp(Schema $schema)
```
Comment (MKodde) [inline app/DoctrineMigrations/Version20180117123823.php]: Cool, didn't know about this feature yet :+1:

File: app/DoctrineMigrations/Version20180117123823.php [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->addSql('ALTER TABLE recipe ADD slug VARCHAR(255) NOT NULL');
```
Comment (MKodde) [inline app/DoctrineMigrations/Version20180117123823.php]: Would it be beneficial to add an index on the slug field?

File: app/DoctrineMigrations/Version20180117123823.php (hunk 2 above)
Comment (pablothedude) [reply app/DoctrineMigrations/Version20180117123823.php]: :thumbsup:

File: src/AppBundle/Repository/RecipeRepository.php (hunk 1 above)
Comment (pablothedude) [reply src/AppBundle/Repository/RecipeRepository.php]: Even aangepast

File: app/DoctrineMigrations/Version20180117123823.php (hunk 3 above)
Comment (pablothedude) [reply app/DoctrineMigrations/Version20180117123823.php]: Nog even toegevoegd

**Pull request merged**